src/ansys/geometry/core/shapes/surfaces/sphere.py

```diff
@@ -197,7 +197,7 @@ def project_point(self, point: Point3D) -> "SphereEvaluation":
         x = origin_to_point.dot(self.dir_x)
         y = origin_to_point.dot(self.dir_y)
         z = origin_to_point.dot(self.dir_z)
-        if np.allclose((x * x + y * y + z * z), Point3D([0, 0, 0])):
+        if np.allclose((x * x + y * y + z * z), 0):
```

Comment:
From numpy documentation:
> Returns True if two arrays are element-wise equal within a tolerance.
Parameters: a, b array_like

It seems that `allclose` uses two arrays. 

I would use the same data type for both values, ideally some form of array.
```suggestion
        if np.allclose(x * x + y * y + z * z, 0):
```
or
```suggestion
        if np.allclose((x * x + y * y + z * z), (0)):
```

Reply:
Oops. Totally right, forgot to remove the parenthesis on the square product. Thanks @AlejandroFernandezLuces